docs/JestObjectAPI.md

```diff
@@ -344,11 +347,19 @@ module.exports = {
 
 Returns the `jest` object for chaining.
 
+### `jest.runAllPromises(runAllTicks)`
```

Comment:
I find it a bit odd to pass in a `runAllTicks` callback, although I agree with trying to avoid a breaking change in `runAllTicks`. I don't have a better suggestion at the moment, so just flagging it 🙂

Reply:
Maybe drop it and document that people should call `runAllTicks` themselves?

Potentially `runAllTicksAndPromises`

Oooor make `runAllTicks` run ticks if fake timers and promises if fake promises?